compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/AMDAIEAIRDmaToAMDAIEDma.cpp

```diff
@@ -0,0 +1,78 @@
+// Copyright 2024 The IREE Authors
+//
+// Licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include "air/Dialect/AIR/AIRDialect.h"
+#include "iree-amd-aie/IR/AMDAIEDialect.h"
+#include "iree-amd-aie/IR/AMDAIEOps.h"
+#include "iree-amd-aie/Transforms/Passes.h"
+#include "mlir/Pass/Pass.h"
+#include "mlir/Support/LogicalResult.h"
+#include "mlir/Transforms/GreedyPatternRewriteDriver.h"
+
+#define DEBUG_TYPE "iree-amdaie-air-dma-to-amdaie-dma"
+
+using namespace xilinx;
```

Comment:
I dont think you need this as we should explicitly call out the xilinx namespace ops so its clear they are coming from mlir-air (which as far as I see you already are).